src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -67,12 +84,21 @@ export default {
 			);
 			this.recommendedLoans = loans;
 		},
-		trackRecommended({ success }) {
-			if (success) this.$kvTrackEvent('loan-card', 'add-to-basket', 'recommended-lending-home');
+		async getMatchedLoans() {
+			const { loans } = await runLoansQuery(
+				this.apollo,
+				{ filterObject: { isMatchable: { eq: true } }, sortBy: 'personalized', pageLimit: 9 },
```

Comment:
Default is `personalized`, so that `sortBy` shouldn't be needed.

Reply:
Right, but this will be updated in other ticket switching to old stack to work in production until matched worked as expected in FLSS

Reply:
Sounds good, but we still might want to hide until the final solution is complete, unless this page isn't public anywhere.

Reply:
oh, yeah, this page isn't public anywhere, is managed by an experiment turned off in production. this is for product QAing on dev